### DISCUSSION TITLE
Keycloak Account console from quickstart repo does not find react.js

### DISCUSSION THREAD

**Discussion by rrodriguezOT** [keycloak/keycloak#26329] [Q&A]
Hi,

Since KC 17 we made an extension for account console v2. We are willing to upgrade KC to 23.

We found some issues that doesn't allow us to move our account theme from KC17 to KC23 and we are working on using keycloak-man example in keycloak-quickstarts repo to build a new theme from it.

We have accomplished to do a build and a deploy without issues but, when the account console is opening, it just stays in the loading loop.

When opening developer tools, the console show an error when trying to get react.js from the next url:

`http://localhost:8080/resources/xiqkr/common/keycloak/web_modules/react.js`

Is there anything we're missing?

Thank you in advance

**Comment by jonkoops** [comment]
The structure of the Account Console has changed, so React is now loaded from a different location. You can [find the URLs](https://github.com/keycloak/keycloak/blob/f20de396aa209e1bea00ce3cf443e85aae3df650/themes/src/main/resources/theme/keycloak.v2/account/index.ftl#L85-L86) in the `index.ftl` template. That said, I would highly recommend that you [try version 3](https://www.keycloak.org/server/features) of the Account Console, which is currently a preview feature, as we're making that the default in Keycloak 24.

**Reply by jonkoops** [reply]
Site-note. We are working on fixing up the 'Keycloak Man' example (https://github.com/keycloak/keycloak-quickstarts/pull/523), and improved documentation is being worked on as well (#26318). 

Feel free to pitch in here if you feel like anything is missing that is relevant for your customization efforts, so we can get them sorted out.

**Comment by rrodriguezOT** [comment]
Ok, I see if v3 is the way we'll follow the changes towards Keycloak 24.

Is there a date when KC24 and the changes to 'Keycloak Man' example and documentation regarding extending account v3 extension will be ready?

**Reply by jonkoops** [reply]
I can't give estimates on things that are in progress, they will be done when we are done. That said, Keycloak 24 is slated for midway February.
